website/scripts/ros/ros_drivers_state.js

```diff
@@ -27,7 +27,7 @@ function subscribeToDriverDiscovery()
         if (GPSStatus == null || GPSStatus == 'undefined')
         return;
 
-        if(message.name == '/hardware_interface/mock_gnss'){
+        if(message.name == '/hardware_interface/novatel_gps_nodelet_wrapper'){
```

Comment:
These names should not be hardcoded as they can change between vehicles. It would be better to use the driver type field instead of the driver name

Reply:
https://github.com/usdot-fhwa-stol/carma-web-ui/commit/9744a445ca2b4964e673675335511d0a1b43de1c

Reply:
Addressed
https://github.com/usdot-fhwa-stol/carma-web-ui/pull/48/commits/294e8c675077209ce78a4c0bd3f5bd5d76936fc7